libs/newsletter/driver/hubspot/src/token/hubspot-forms.token.ts

```diff
@@ -8,13 +8,16 @@ import { Title } from '@angular/platform-browser';
 import { Router } from '@angular/router';
 
 import {
-  DaffHubspotFormsService,
   daffHubspotFormsServiceFactory,
+  DaffHubspotFormsInterface,
 } from '@daffodil/driver/hubspot';
 
 import { DaffNewsletterConfigToken } from '../config/newsletter-config.interface';
 
-export const DAFF_NEWSLETTER_HUBSPOT_FORMS_TOKEN = new InjectionToken<DaffHubspotFormsService>('DAFF_NEWSLETTER_HUBSPOT_FORMS_TOKEN',
+/**
+ * A token for combining the inputs to the HubspotFormsServiceFactory into a single dependency.
+ */
+export const DAFF_NEWSLETTER_HUBSPOT_FORMS_TOKEN: InjectionToken<DaffHubspotFormsInterface> = new InjectionToken<DaffHubspotFormsInterface>('DAFF_NEWSLETTER_HUBSPOT_FORMS_TOKEN',
```

Comment:
Why is this doc different than the `contact` version of the same token? Can you give an example usage in the doc here to clarify how one would use it?